elide-core/src/main/java/com/yahoo/elide/resources/JsonApiEndpoint.java

```diff
@@ -121,7 +121,10 @@ public Response delete(
         return build(elide.delete(path, jsonApiDocument, getUser.apply(securityContext)));
     }
 
-    private static Response build(ElideResponse response) {
-        return Response.status(response.getResponseCode()).entity(response.getBody()).build();
+    protected Response build(ElideResponse response) {
```

Comment:
Where does the `failureReason` get used? I see it being set, but never logged or added to a (presumably verbose?) response.

Reply:
Or is the plan that arbitrary endpoints could leverage this functionality?

Reply:
We'll be using it internally. Right now error cause is opaque to consumers of Elide unless you override a bunch of stuff.